src/services/doppler-legacy-client.ts

```diff
@@ -251,6 +252,10 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
         return { expectedError: { accountNotValidated: true } };
       }
 
+      if (!response.data.success && response.data.error == 'AccountIsCanceled') {
```

Comment:
This code seems to be wrong, did you tested it? please check [the ticket|http://jira.makingsense.com/browse/DBR-192] and fix it if the information is wrong.

![image](https://user-images.githubusercontent.com/1157864/57153728-6bfca500-6dad-11e9-8e74-4350f5dff304.png)

Reply:
I took this from `ApplicationErrorCode.cs`

![image](https://user-images.githubusercontent.com/10213170/57153864-c138b680-6dad-11e9-8f76-02cbaa9af78b.png)

Reply:
Ok, probalo y en todo caso actualizá el ticket por favor.

Reply:
You were right, I have to use CancelatedAccount 😅